### PR TITLE
UI: Update and Add mouseover dialog

### PR DESCRIPTION
**Pull request by TheTechnician27**
### Description of Changes
- GraphicsSettingsWidget.cpp: Add mouseover dialog for Internal Resolution, Vertical Stretch, Cropping, TV Shader, and GS Dump Compression options.
- Minor typographical fixes in other files.

### Rationale behind Changes
Mouseover dialog is missing in a number of graphical options in the Qt interface. As Internal Resolution is likely to be one of the first settings a new user will change, this may be useful to them.

### CODE REVIEW

**Comment by TheTechnician27** [comment]
I think I've scoured things pretty thoroughly, and I've added all the dialog boxes that I feel wouldn't be left to someone more knowledgable, so I won't be adding any more commits to this merge.

**Comment by refractionpcsx2** [comment]
I'll give it ~24 hours jusst in case you change your mind :P

**Comment by RedDevilus** [comment]
Relatable moment haha.

File: pcsx2-qt/Settings/SettingsDialog.cpp [hunk 1]
```diff
@@ -115,7 +115,7 @@ void SettingsDialog::setupUi(const GameList::Entry* game)
 	{
 		addWidget(m_game_fix_settings_widget = new GameFixSettingsWidget(this, m_ui.settingsContainer), tr("Game Fix"),
 			QStringLiteral("close-line"),
-			tr("<strong>Game Fix Settings</strong><hr>Gamefixes can work around incorrect emulation in some titles<br>however they can "
+			tr("<strong>Game Fix Settings</strong><hr>Gamefixes can work around incorrect emulation in some titles.<br>However, they can "
```
Comment (Iniquitatis) [inline pcsx2-qt/Settings/SettingsDialog.cpp:118]:
```suggestion
			tr("<strong>Game Fix Settings</strong><hr>Game fixes can work around incorrect emulation in some titles.<br>However, they can "
```

Then maybe also this? Or vice versa? There are both "gamefix" and "game fix" on a single line. More consistensy wouldn't hurt.

**Comment by IlDucci** [comment]
Sorry if this feels like hijacking, but here's some stuff that could be changed for consistency:
 - Ending periods at the end of each tooltip. Some strings have them, some strings do not.
 - fast forward/fast-forward/fastforward, frame buffer/framebuffer, the aforementioned gamefix/game fix/game fixes/gamefixes, multi-threaded/multithreaded, memory card (whether to title uppercase or not), savestate/save state, slow motion/slow-motion/
slowmotion, swap chain/swapchain. I wrote this list when I did a translation pass of 1.7.

**Comment by TheTechnician27** [comment]
@IlDucci I definitely agree that more consistency wouldn't hurt. For the time being, that's not what I was going for, but it should definitely be done in the future. Just as another example, the option is 'De-interlacing' while the mouseover dialog is 'Deinterlacing'.

**Comment by refractionpcsx2** [comment]
Why couldn't you all have made these suggestions BEFORE I merged it? I left it here for a week on purpose so any extra changes could have been made...

**Comment by IlDucci** [comment]
Sorry about that, I saw this post because of the latest reply.

**Comment by TheTechnician27** [comment]
> Why couldn't you all have made these suggestions BEFORE I merged it? I left it here for a week on purpose so any extra changes could have been made...

It's all good. I intentionally didn't do any consistency changes (except 'Clut' -> 'CLUT' because that's just objectively correct) because it seems like a waste of effort until all of the mouseover dialog is written (or at least until a solid third of the options aren't blank). Once all or most of the dialog is filled in, it seems like then would be a good time to just do the consistency changes all in one go instead of having them spread out across two or more PRs for no reason.

I was aware of these issues, so them having been brought to my attention likely wouldn't have changed the PR.

**Pull request merged**